src/_gettsim/transfers/rente.py

```diff
@@ -895,22 +1161,54 @@ def ges_rente_vorauss_frauen(
         weiblich
         and ges_rente_wartezeit_15
         and y_pflichtbeitr_ab_40 > ges_rente_params["rente_für_frauen_pflichtbeitr_y"]
-        and geburtsjahr < ges_rente_params["abolishment_cohort_rente_für_frauen"]
+        and geburtsjahr < ges_rente_params["first_birthyear_without_rente_für_frauen"]
     )
 
     return out
 
 
-def _ges_rente_vorauss_arbeitsl(
+@dates_active(end="2011-12-31", change_name="_ges_rente_vorauss_arbeitsl")
+def _ges_rente_vorauss_arbeitsl_ohne_2012_reform(
+    arbeitsl_1y_past_585: bool,
+    ges_rente_wartezeit_15: bool,
+    pflichtbeitr_8_in_10: bool,
+) -> bool:
+    """Eligibility for Altersrente für Arbeitslose (pension
+    for unemployed. Wartezeit 15 years, 8 contributionyears past 10 years, being
+    at least 1 year unemployed after age 58 and 6 months.
+
+    Parameters
+    ----------
+    arbeitsl_1y_past_585
+        See basic input variable
+        :ref:`arbeitsl_1y_past_585 <arbeitsl_1y_past_585>`.
+    ges_rente_wartezeit_15
+        See :func:`ges_rente_wartezeit_15`
+    pflichtbeitr_8_in_10
+        See basic input variable :ref:`pflichtbeitr_8_in_10 <pflichtbeitr_8_in_10>`.
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+
+    out = arbeitsl_1y_past_585 and ges_rente_wartezeit_15 and pflichtbeitr_8_in_10
+
+    return out
+
+
+@dates_active(start="2012-01-01", change_name="_ges_rente_vorauss_arbeitsl")
```

Comment:
See elsewhere, reform was in 2007, forward-looking agents should have worked with that?

`dates_active` should have an end date -- 1951 + x where x is the relevant retirement age (65 or whatever)